frontend/viewdir/template.njk

```diff
@@ -12,7 +12,7 @@
     {% endblock %}
 </head> 
 
-<body>
+<main>
```

Comment:
You should still have a `<body>` tag in the resultant html. `<main>` just indicates which part of the `<body>` is holding the primary content of the page.
```
<body>
  <header><!-- --></header>
  <main><!-- --></main>
  <footer><!-- --></footer>
</body>
```